doc/guides/kconfig/index.rst

```diff
@@ -848,15 +848,17 @@ Kconfig Functions
 *****************
 
 Kconfiglib provides user-defined preprocessor functions that
-we use in Zephyr to expose devicetree information to Kconfig.
+we use in Zephyr to expose devicetree or cmake environment information to
+Kconfig.
 For example, we can get the default value for a Kconfig symbol
 from the devicetree.
+See the Python docstrings in ``scripts/kconfig/kconfigfunctions.py`` for more
```

Comment:
OK for now, but we can also use the sphinx autodoc capabilities (as done for west).